src/HealthChecks.AzureServiceBus/AzureServiceBusQueueMessageCountThresholdHealthCheck.cs

```diff
@@ -0,0 +1,52 @@
+using HealthChecks.AzureServiceBus.Configuration;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace HealthChecks.AzureServiceBus;
+
+public class AzureServiceBusQueueMessageCountThresholdHealthCheck : AzureServiceBusHealthCheck<AzureServiceBusQueueMessagesCountThresholdHealthCheckOptions>, IHealthCheck
```

Comment:
I would merge that check with `AzureServiceBusDeadLetterQueueMessageCountThresholdHealthCheck ` into single one and play with options object to set all 4 thresholds:
```
public struct Threshold
{
public int Degraded {get;}
public int Unhealthy {get;}
}

public AzureServiceBusQueueMessagesCountThresholdHealthCheckOptions 
{
   public Threshold ActiveMessages {get; set; }
   public Threshold DeadLetterMessages { get; set; }
}
```

Reply:
It's been changed